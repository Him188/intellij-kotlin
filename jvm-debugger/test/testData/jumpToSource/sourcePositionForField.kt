package sourcePositionForField

abstract class AbstractClass {
    val aInt = 10
}

class TestClass : AbstractClass() {
    val myInt = 20
}

fun main() {
    val instance = TestClass()
    // JUMP TO SOURCE: instance.myInt
    // JUMP TO SOURCE: instance.aInt
    //Breakpoint!
    println("")
}
