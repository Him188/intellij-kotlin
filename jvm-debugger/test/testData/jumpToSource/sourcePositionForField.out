LineBreakpoint created at sourcePositionForField.kt:16
Run Java
Connected to the target VM
sourcePositionForField.kt:16
Source position of instance.myInt is 8
Source position of instance.aInt is 4
Disconnected from the target VM

Process finished with exit code 0
